gaptrain/gap.py

```diff
@@ -607,9 +621,9 @@ def ase_gap_potential_str(self,
         here = os.path.abspath(os.path.dirname(__file__))
         pt = open(os.path.join(here, 'iicalculator.py'), 'r').readlines()
 
-        pt += [f'inter_gap = quippy.Potential("IP GAP", '
+        pt += [f'inter_gap = {potential_class}("IP GAP", '
```

Comment:
Probably fine, but some string lines have \n and some don't

Reply:
mehh